pkgs/development/tools/misc/catppuccin-catwalk/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, python3
+,
+}: python3.pkgs.buildPythonApplication rec {
+  pname = "catppuccin-catwalk";
+  version = "0.4.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "catppuccin_catwalk";
+    hash = "sha256-5TAw5H3soxe9vLhfj1qs8uMr4ybrHlCj4zdsMzvPo6s=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pillow
+  ];
+
+  nativeBuildInputs = with python3.pkgs; [
+    poetry-core
+  ];
```

```suggestion
  nativeBuildInputs = with python3.pkgs; [
    poetry-core
  ];

  propagatedBuildInputs = with python3.pkgs; [
    pillow
  ];
```
Usually, `nativeBuildInputs` come first.